index.html

```diff
@@ -4,6 +4,12 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>JavaScript Foundations</title>
+    <script defer src="./js/exercise-00.js"></script>
+    <script defer src="./js/exercise-01.js"></script>
+    <script defer src="./js/exercise-02.js"></script>
+    <script defer src="./js/exercise-03.js"></script>
+    <script defer src="./js/exercise-04.js"></script>
   </head>
-  <body></body>
+  <body>
```

Comment:
- [ ] Add content to body. Otherwise Netlify will not run any deploy in preview environment.